GOVERNANCE.md

```diff
@@ -0,0 +1,126 @@
+  key discussion points and decisions and copy those to GitHub.
+* Develop a proposed resolution to the issue (along the lines of an ISO
+  "Technical Report").
+* Return with a proposal (text and diagrams) for resolution of the issue (along
+  the lines of an ISO "Technical Spec"). Ideally this proposal is in a form that
+  can be easily: a) turned into a PR (for a GitHub document), or b)
+  copy-and-pasted as a revision to a Google doc or other format.
+
+### Closure
+
+* If the Editors believe an issue has been resolved via one or more PRs that
+  have been accepted and merged, then one of the Editors should apply the label
+  status: PR-merged and close the issue. If the Editors believe consensus has
+  been achieved about some other resolution of the issue — and that resolution
+  is fully documented in the issue — then one of the Editors should apply the
+  label `status: last-call`.
+  * Once that label has been applied, a group member MUST object to closure by
+    making a comment on the issue within 5 calendar days to reopen discussion of
+    the issue.
+  * If there is no objection within 5 calendar days, the proposed resolution
+    shall be applied to the deliverable by one of the Maintainers and one of the
+    Editors shall close the issue with no further discussion.
+  * If there is an objection, the Editors will take it to a group meeting to
+    reach final consensus on closure.
+
+### Editor Activities
+
+* Labelling issues on weekly review.
+* Prioritizing issues.
+* Finding the appropriate `assignee` to work on an issue.
+* Create issues that are required for advancement of the repository.
+
+## Maintainers
+
+* Maintainers have write access to GitHub repositories and act as project
+  administrators. They approve and merge pull requests, cut releases, and guide
+  collaboration with the community. They have:
+  * Ability to merge PR's
+  * Commit access to their project's repositories
+* Both maintainers and non-maintainers may propose changes to source code. The
+mechanism to propose such a change is a GitHub pull request. Maintainers review
+and merge (land) pull requests.
+* The Editors should in turn appoint a set of Maintainers who have the Github
+  skills (and the necessary permissions) to accept PRs and publish versions of
+  the deliverable. Typically a subset of the Editors serve as Maintainers, but
+  all the Editors can serve in this role, or it can be assigned to others in the
+  group.
+* If a maintainer opposes a proposed change, then the change cannot land. The
+  exception is if the Governance Committee ( GC) votes to approve the change
+  despite the opposition. Usually, involving the GC is unnecessary.
+
+### Maintainer activities
+
+* Helping users and novice contributors
+* Contributing code and documentation changes that improve the project
+* Reviewing and commenting on issues and pull requests
+* Participation in working groups
+* Merging pull requests
+
+## Administrators
```

Comment:
Needs review

Reply:
I think we need to shorten & simplify so as to not overwhelm potential contributors. The ToIP group-related stuff should be in ToIP wiki. This text should focus on the contribution for Tech architecture.